.github/workflows/unit-test.yml

```
@@ -0,0 +1,57 @@
+
+name: Unit tests
+
+on:
+  pull_request:
+    paths:
+    - ".github/workflows/unit-test.yml"
+    - "**/*.py"
+
+permissions:
+  contents: read
+  pull-requests: write
+
+env:
+  POSTGRES_VERSION: "15"
+  POSTGIS_VERSION: "3.0"
```

Autant utiliser la dernière version non ?

```suggestion
  POSTGIS_VERSION: "3.3"
```